outline.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Serverless perspectives
+subtitle: An outline 
+...
+
+I.  What is serverless?
+    A.  Elucidating tools, patterns, and practices
+        1. Functions as a Service
+            a. Variable cost application models
+        2. Utility computing
+        3. Frameworks
+            a. Serverless
+            b. Others?
+
+II.  Why should I use serverless?
+    A.  Helping you do what you know you should be doing already
+        1. API-driven development
+            a. Platforms as a first-class citizen
```

Comment:
"Platform" here means a cloud provider (maybe stuck in recent lingo where "Platform" = "AWS"), or the idea that serverless dev should follow a platform-like approach?